### PR TITLE
expense_meter PR

### PR DESCRIPTION
**Pull request by kkasaija**
### In this PR, I did the following
[walk thru demo link](https://drive.google.com/file/d/1pzzDw8MJDa-5DeA1Ckkk7n556VVpWSWE/view?usp=sharing)
- Created a web-based mobile app with the following features
- [x] Splash screen: A simple page with the name 'expense meter' and links to the signup and log-in pages.
- [X] Sign up: The user is able to register in the app with full name, email and password (all mandatory).
- [x] Login: The user can log into the app using email and password. If the user is not logged in, they can't access pages.
- [x] When the user logs in, they are presented with the categories page.
- [x] Transactions page: For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page, the user could see the total amount for the category (the sum of all of the amounts of the transactions in that category).
- [x] There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.
- [x] Unit and integration tests for all the most important components of your RoR application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,230 @@
...
+- **CRUD Operations on all resources with authorization roles**
+- **Test cases using Rspec and Capybara**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a> 
+
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please deploy your application and add a link in the README

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Kasaija-Kenneth, 

Good job so far! There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (kkasaija) [reply README.md]: Noted @DeliceLydia with thanks. application deployed and demo link added

**Review by mikethreels — APPROVED** [review]
Hi Ken, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkasaija** [comment]
Much obliged @mikethreels 🙏🙏🙏🙏

**Pull request merged**